Catappult/appcoins-wallet-android
# [APPC-2170] Subscription listing

**What does this PR do?**

Implements the API and adds the necessary changes to the UI to match the new API

**Database changed?**

   No

**Where should the reviewer start?**

SubscriptionListFragment.kt

**How should this be manually tested?**

Use mocked subs to test how it behaves UI wise with many subs
Do a sub on trivial drive, check if it then shows up on the list, then cancel the subscription and check if it shows up on the list as canceled (with the expires on instead of the price)

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-2170

**Code Review Checklist**
